subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy

```diff
@@ -86,4 +90,13 @@ class RepoScriptBlockUtil {
             """
         }
     }
+
+    static String googleRepositoryDefinition() {
+        String repoUrl = System.getProperty('org.gradle.integtest.mirrors.google')
+        if (repoUrl) {
+            return "maven { name 'Google'; url '${repoUrl}' }"
```

Comment:
:o: I guess we could just use a multi-line String. Might make it more readable.

Reply:
Fixed